frontend/ui/widget/bookmapwidget.lua

```diff
@@ -1018,29 +1019,144 @@ function BookMapWidget:update()
+                align = "left",
+            },
+            {
+                text = "\u{2796}", -- Heavy minus sign
+                enabled_func = function() return self.toc_depth > 0 end,
+                callback = function()
+                    if self:updateTocDepth(self.flat_map and 1 or -1, nil) then
+                        self:update()
+                    end
+                end,
+                width = plus_minus_width,
+            },
+            {
+                text = "\u{2795}", -- Heavy plus sign
+                enabled_func = function() return self.toc_depth < self.max_toc_depth end,
+                callback = function()
+                    if self:updateTocDepth(self.flat_map and -1 or 1, nil) then
+                        self:update()
+                    end
+                end,
+                width = plus_minus_width,
+            }
+        },
+        {
+            {
+                text = _("Page slots width"),
+                callback = function() end,
+                align = "left",
+                -- Below, minus increases page per row and plus decreases it.
+                -- It feels more natural this way: + will make everything (page slots and the grid) bigger.
+            },
+            {
+                text = "\u{2796}", -- Heavy minus sign
+                enabled_func = function() return self.pages_per_row < self.max_pages_per_row end,
+                callback = function()
+                    if self:updatePagesPerRow(10, true) then
+                        self:update()
+                    end
+                end,
+                width = plus_minus_width,
+            },
+            {
+                text = "\u{2795}", -- Heavy plus sign
+                enabled_func = function() return self.pages_per_row > self.min_pages_per_row end,
+                callback = function()
+                    if self:updatePagesPerRow(-10, true) then
+                        self:update()
+                    end
+                end,
+                width = plus_minus_width,
+            }
+        },
+    }
+    button_dialog = ButtonDialog:new{
+        -- width = math.floor(Screen:getWidth() / 2),
+        width = math.floor(Screen:getWidth() * 0.9), -- max width, will get smaller
+        shrink_unneeded_width = true,
+        buttons = buttons,
+        anchor = function()
+            return self.title_bar.left_button.image.dimen
+        end,
+    }
+    UIManager:show(button_dialog)
+end
+function BookMapWidget:showAbout()
     UIManager:show(InfoMessage:new{
         text = _([[
 Book map displays an overview of the book content.
 
 If statistics are enabled, black bars are shown for already read pages (gray for pages read in the current reading session). Their heights vary with the time spent reading the page.
-Chapters are shown above their pages.
-Under the pages can be found some indicators:
+Chapters are shown above the pages they span.
```

Comment:
```
span (v)
verb (used with object), spanned, span·ning.
to measure by the hand with the thumb and little finger extended.
to encircle with the hand or hands, as the waist.
to extend over or across (a section of land, a river, etc.).
to provide with something that extends over: to span a river with a bridge.
to extend or reach over (space or time): a memory that spans 90 years.
Mathematics. to function (in a subspace of a vector space) as a span.
Archery. to bend (the bow) in preparation for shooting.
```
looked ok to me :)
But ok with encompass.

Reply:
That's what it means I guess, chapters (titles) are shown above the pages (slots) they span/encompass:
![image](https://user-images.githubusercontent.com/24273478/226971287-a727f68f-79ca-4374-a8e7-9698ba0668c8.png)

Reply:
Yes, they do span it quite literally. :-)

A chapter does span multiple pages of course, but somehow it just seems a bit weird to me to put it that way. I think perhaps because it's the opposite that's remarkable, a chapter that *doesn't* span multiple pages.

Another possible word, maybe *contain*?